.gitignore

```diff
@@ -5,4 +5,5 @@ ropsten_pass.txt
 PRIVATE_README.md
 
 *.log*
-*sublime*
\ No newline at end of file
+*sublime*
+yarn*
```

Comment:
why hiding yarn stuff? to push us to use npm?

Reply:
yea, we decided `npm` was more friendly to newcomers, right? (you can still use `yarn` if you want)